packages/wallets/src/AppContent.scss

```diff
@@ -6,10 +6,11 @@
     width: 100%;
     align-self: stretch;
     background-color: var(--general-main-1, #ffffff);
-    min-height: calc(100vh - 8.4rem); // 100vh - (4.8rem header + 3.6rem footer)
+    height: calc(100vh - 8.4rem); // 100vh - (4.8rem header + 3.6rem footer)
+    /* stylelint-disable-next-line declaration-no-important */
+    overflow-y: auto !important; //override overflow: initial from core app content
 
     @include mobile {
-        overflow-y: auto;
         height: calc(var(--wallets-vh, 1vh) * 100 - 4rem); // 100vh - 4rem (header)
```

Comment:
`overflow-y: auto` added for both desktop and mobile instead of just mobile to fix issue where screen cannot be scrolled after switching language due to `overflow: initial` from core's app content (issue shown below):

https://github.com/user-attachments/assets/a83c76a9-48b5-4c0a-92fa-16331d545308

Additionally `min-height` prop is changed to `height` to fix issue where onboarding content can't be seen due to the overflow set (issue shown below):

https://github.com/user-attachments/assets/4a75aebf-8037-410e-a614-61c416d743ba